### PR TITLE
Added support to check for CRDs

### PR DESCRIPTION
**Pull request by nveenjain**
Signed-off-by: Naveen Jain <nveenjain@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # learn-layer5
 
+# To check for smi conformance of a deployed service mesh
+We use kuttl to check for SMI conformance. All the tests are writtten in smi-test directory of this repository.
+Execute the following command to run the smi-conformace tests:-
+
+```shell
+kubectl kuttl test  --skip-cluster-delete=true --start-kind=false ./smi-test
```
Comment (leecalcote) [inline README.md:8]: Can we instead use the kuttl `harness` and `testutils` packages? - https://kuttl.dev/docs/api-integration.html

File: smi-test/trafficAccess/01-assert.yaml [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    matchLabels:
+      app: app-c
+  template:
+    metadata:
+      labels:
+        app: app-c
+    spec:
+      containers:
+      - name: app-c
+        image: layer5/sample-app-service:dev
```
Comment (leecalcote) [inline smi-test/trafficAccess/01-assert.yaml:75]: _Note: We can incorporate kustomize to override these values (like the image name) later._

File: smi-test/trafficAccess/execThis.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+echo $(kubectl get service service-a --namespace=$NAMESPACE -o jsonpath="{.spec.ports[0].nodePort}")
+kubectl describe pods --namespace=$NAMESPACE
+curl --location --request POST 'http://localhost:'$(echo $(kubectl get service service-a --namespace=$NAMESPACE -o jsonpath="{.spec.ports[0].nodePort}"))'/call' -w "%{http_code}" --data-raw '{"host": "http://service-b/post",}'
```
Comment (leecalcote) [inline smi-test/trafficAccess/execThis.sh:4]: Can we displace this with use of one of Meshery's request generators (e.g. fortio)?

File: README.md (hunk 1 above)
Comment (nveenjain) [reply README.md:8]: Yeah, the same tests can be invoked using golang apis...

File: smi-test/trafficAccess/execThis.sh (hunk 3 above)
Comment (leecalcote) [reply smi-test/trafficAccess/execThis.sh:4]: Cc @kushthedude

**Comment by leecalcote** [comment]
@nveenjain good to merge, so that you can advance. We'll just want to take note (make an issue or incorporate into the design spec) of the unresolved discussions above.

**Comment by nveenjain** [comment]
@leecalcote please /hold thus PR for now, I'll make some changes in it(working on a few)... And maybe that would resolve most of discussions...

**Pull request merged**